semgrep/semgrep/core_runner.py

```diff
@@ -273,11 +273,7 @@ def _run_rules_direct_to_semgrep_core(
                     debug_tqdm_write(f"Running rule {rule.id}...")
                     with tempfile.NamedTemporaryFile(
                         "w", suffix=".yaml"
-                    ) as rule_file, tempfile.NamedTemporaryFile(
-                        "w"
-                    ) as target_file, tempfile.NamedTemporaryFile(
-                        "w"
-                    ) as equiv_file:
+                    ) as rule_file, tempfile.NamedTemporaryFile("w") as target_file:
```

Comment:
Unrelated - without `equivalences` we no longer need `equiv_file`. No need to open another file every run.